stop_line_list:
  -
    id : 1
    tl_id : 1
    pose :
      x : 12.0
      y : 57.0
      z : 0.0